gdnative-core/src/class.rs

```diff
@@ -279,6 +309,17 @@ where
     }
 }
 
+impl<T> FromVariant for Instance<T>
```

Comment:
This is very cool, that means custom script types can be used as parameters! Very nice :ok_hand: